nUpdate/Updating/UpdateResult.cs

```diff
@@ -4,19 +4,24 @@
 using System.Collections.Generic;
 using System.Linq;
 using nUpdate.Core;
+using nUpdate.UpdateEventArgs;
 
 namespace nUpdate.Updating
 {
     internal class UpdateResult
     {
         private UpdateConfiguration _newestConfiguration;
-        private readonly List<UpdateConfiguration> _newUpdateConfigurations = new List<UpdateConfiguration>();
-        private readonly bool _updatesFound;
+        private List<UpdateConfiguration> _newUpdateConfigurations = new List<UpdateConfiguration>();
+        private bool _updatesFound;
 
         /// <summary>
         ///     Initializes a new instance of the <see cref="UpdateResult" /> class.
         /// </summary>
-        public UpdateResult(IEnumerable<UpdateConfiguration> packageConfigurations, UpdateVersion currentVersion,
+        public UpdateResult()
+        {
+        }
+
+        public void Init(IEnumerable<UpdateConfiguration> packageConfigurations, UpdateVersion currentVersion,
```

Comment:
As written above, use a **Dictionary** instead of events and do that stuff in the constructor.